Week8/MovieViewer/app/build.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'kotlinx-serialization'
+
+def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystoreProperties = new Properties()
+keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+
+android {
+    compileSdkVersion 30
```

Comment:
I'd say avoid using the API 30 (Android 11) as it's in the beta phase, and this might cause tons of problems for you, that we won't know are happening.